trelloscrum.js

```diff
@@ -257,7 +257,7 @@
     }
 
     //track card-titles
-    new TextChangeListener("a", "a.js-card-name", function (elm) {
+    new TextChangeListener("span", "span.js-card-name", function (elm) {
```

Comment:
Yes. I didn't review the current design, but it seems that due to performance reasons the listener must depend on a specifiek element.